Description: mail command absolute pathname
 This patch changes the default command to /usr/bin/mail in various
  configuration files and manual pages.                            
Bug-Debian: http://bugs.debian.org/588862
Forwarded: not-needed
Last-Update: 2018-10-01
Author: Gennaro Oliva <oliva.g@na.icar.cnr.it>


diff --git a/doc/man/man1/strigger.1 b/doc/man/man1/strigger.1
index 3c74d8a..97f0d37 100644
--- a/doc/man/man1/strigger.1
+++ b/doc/man/man1/strigger.1
@@ -373,7 +373,7 @@ $ cat /usr/sbin/primary_slurmctld_failure
 strigger \-\-set \-\-primary_slurmctld_failure \\
          \-\-program=/usr/sbin/primary_slurmctld_failure
 # Notify the administrator of the failure using e\-mail
-/bin/mail slurm_admin@site.com \-s Primary_SLURMCTLD_FAILURE
+/usr/bin/mail slurm_admin@site.com \-s Primary_SLURMCTLD_FAILURE
 
 $ strigger \-\-set \-\-primary_slurmctld_failure \\
            \-\-program=/usr/sbin/primary_slurmctld_failure
@@ -392,7 +392,7 @@ $ cat /usr/sbin/slurm_admin_notify
 strigger \-\-set \-\-node \-\-down \\
          \-\-program=/usr/sbin/slurm_admin_notify
 # Notify administrator using by e\-mail
-/bin/mail slurm_admin@site.com \-s NodesDown:$*
+/usr/bin/mail slurm_admin@site.com \-s NodesDown:$*
 
 $ strigger \-\-set \-\-node \-\-down \\
            \-\-program=/usr/sbin/slurm_admin_notify
